# why the sub Process couldn't access its data, in python
start()
